en/docs/includes/db-advanced-config.md

```diff
@@ -24,5 +25,6 @@ Apart from the basic configurations specified above, WSO2 Identity Server suppor
 	validationQuery="SELECT 1"
 	validationInterval="30000"
 	defaultAutoCommit=false
+    commitOnReturn=true
```

Comment:
Again, as said previously, there is a slight indentation issue.

Reply:
Fixed with https://github.com/wso2/docs-is/pull/3374/commits/4c1d783e8481c394fdcca61b2cf9bfd8b5293a85.